src/offline/component.js

```diff
@@ -508,8 +508,10 @@ export const Controller = class {
    */
   removeZoomConstraints_() {
     const view = this.map.getView();
-    view.setMaxZoom(this.originalMaxZoom);
-    view.setMinZoom(this.originalMinZoom);
+    if (this.originalMaxZoom) {
```

Comment:
```suggestion
    if (this.originalMaxZoom !== undefined) {
```
Otherwise, 0 would be true.
Also, it is possible to have a minZoom but not a maxZoom ? In that's the  case, the test is bad.

Reply:
I don't think it is possible, but I modified the condition to make sure both are undefined.